tests/observability/metrics/test_vms_metrics.py

```diff
@@ -471,3 +471,14 @@ def test_kubevirt_vmi_non_evictable(
             metric_name="kubevirt_vmi_non_evictable",
             expected_value="1",
         )
+
+
+class TestOutdatedVmi:
+    @pytest.mark.post_upgrade
+    @pytest.mark.polarion("CNV-11749")
+    def test_metric_kubevirt_vmi_number_of_outdated(self, prometheus, kubevirt_resource):
+        validate_metrics_value(
+            prometheus=prometheus,
+            metric_name="kubevirt_vmi_number_of_outdated",
+            expected_value=kubevirt_resource.instance.status.outdatedVirtualMachineInstanceWorkloads,
+        )
```

Comment:
This test should be in upgrade lane. 1) before upgrade value should be 0 and after upgrade the value should be non-zero.